<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="amtortextBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>571</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="amtorFreqCorrection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="amtorStrengthMeter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="amtorAfcon">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>afc off</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>afc on</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="amtorReverse">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>reverse</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="amtorFecError">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>non strict</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>strict</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="amtorMessage">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>all text</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>message</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>amtor</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
